portalnet/src/storage.rs

```diff
@@ -27,6 +27,9 @@ use trin_types::content_key::{ContentKeyError, HistoryContentKey, OverlayContent
 use trin_types::distance::{Distance, Metric, XorMetric};
 use trin_utils::bytes::{hex_decode, hex_encode, ByteUtilsError};
 
+const BYTES_IN_MB_U64: u64 = 1000 * 1000;
+const BYTES_IN_MB_F64: f64 = 1000.0 * 1000.0;
```

Comment:
If we use numbers like 1024 here we end up squeaking out just a little extra storage from each user....

Reply:
I'm not opposed to that

Reply:
after some discussion with the team, I opted to keep the multiples of 1000. a higher default is an alternative to elicit more storage from the median user